File: src/main/java/com/lmx/springbootmybatis/com/lmx/StudentServce.java
package com.lmx.springbootmybatis.com.lmx;

import com.lmx.springbootmybatis.com.lmx.po.Student;
import org.apache.ibatis.annotations.Delete;
import org.apache.ibatis.annotations.Insert;
import org.apache.ibatis.annotations.Select;
import org.apache.ibatis.annotations.Update;

import java.util.List;

/**
 * @author LiumengMengXiang
 * @create 2019-04-11:22
 */
public interface StudentServce {
    List<Student> findall();

    Student findByid(int id);

    boolean insert(Student student);

    boolean update(Student student);

    boolean delete(int id);
}

File: src/main/java/com/lmx/springbootmybatis/com/lmx/StudentServce.kt
package com.lmx.springbootmybatis.com.lmx

import com.lmx.springbootmybatis.com.lmx.po.Student
import org.apache.ibatis.annotations.Delete
import org.apache.ibatis.annotations.Insert
import org.apache.ibatis.annotations.Select
import org.apache.ibatis.annotations.Update

/**
 * @author LiumengMengXiang
 * @create 2019-04-11:22
 */
interface StudentServce {
    fun findall(): List<Student>

    fun findByid(id: Int): Student

    fun insert(student: Student): Boolean

    fun update(student: Student): Boolean

    fun delete(id: Int): Boolean
}
